src/ui/pages/work/WorkorderScheduling.js

```diff
@@ -23,7 +24,27 @@ const WorkorderScheduling = (props) => {
 
     return (
         <div style={{width: "100%", marginTop: 0}}>
+            <Grid container justify="space-between" spacing={12} style={{margin: "0px"}}>
```

Comment:
Use:
```js
 <Grid container justify="space-between" spacing={2}>
```
The style for the margin is not needed and a spacing of 2 units should be enough. Also, the API for the Grid component specifies that the spacing prop ranges between 1 and 10 units: https://material-ui.com/api/grid/